samples-lib/src/main/java/com/esri/arcgismaps/sample/sampleslib/components/BottomSheet.kt

```diff
@@ -64,7 +64,7 @@ fun BottomSheet(
     Box(
         modifier = Modifier
             .fillMaxSize()
-            .safeContentPadding()
+            .systemBarsPadding()
```

Comment:
I find `systemBarsPadding` to be a better choice than `safeContentPadding` for the samples-lib bottom sheet.